codespell_lib/data/dictionary.txt

```diff
@@ -5036,6 +5036,10 @@ ciontrol->control
 ciper->cipher
 cipers->ciphers
 cipherntext->ciphertext
+ciphersute->ciphersuite
+ciphersutes->ciphersuites
```

Comment:
Could maybe also add
```
ciphersuit->ciphersuite
ciphersuits->ciphersuites
```
?

And perhaps it's also worth adding
```
cyphersuite->ciphersuite
cyphersuites->ciphersuites
```
:man_shrugging:

Reply:
Done.